### PR TITLE
Feat/like functionality

### PR DESCRIPTION
**Pull request by izabelladobielewska**
In this US I added a like button to the card, a like icon to nav, I created a page with likes, where the liked cards are displayed. the likes are stored in a local storage. 
Thank you for reviewing! 🙏

### CODE REVIEW

File: components/Card/index.js [hunk 1]
```diff
@@ -2,8 +2,9 @@ import styled from "styled-components";
 import Rating from "../Rating";
 import AverageRating from "../AverageRating";
 import { useState, useEffect } from "react";
+import LikeButton from "../LikeButton";
```
Comment (milodevhh) [inline components/Card/index.js:5]: Well done, that you use a component for that.

File: components/Card/index.js [hunk 2]
```diff
@@ -33,36 +34,83 @@ export default function Card({ card, mutateCards }) {
...
+`;
+const Likes = styled.div`
+  grid-column: 3;
+  place-self: center;
+  padding: 1rem;
+`;
+
 const Article = styled.article`
-  box-shadow: 0px 0px 10px #ccc;
+  box-shadow: 7px 7px 0px #f3c3a8;
```
Comment (milodevhh) [inline components/Card/index.js:98]: Nice color pick, I like that.

File: components/Header/index.js [hunk 3]
```diff
@@ -4,9 +4,17 @@ import { PreferencesIcon } from "@/public/assets/images/preferences-icon";
...
 import { AddIcon } from "@/public/assets/images/add-icon";
+import cards from "@/public/assets/cards";
+import { use } from "react";
+import { useRouter } from "next/router";
 
 export default function Header() {
   const pathName = usePathname();
+  const router = useRouter();
+  function handleShowLikes() {
+    router.push("/", "likes");
```
Comment (milodevhh) [inline components/Header/index.js]: best way to do so, I used it as well.

**Review by milodevhh — APPROVED** [review]
Hey Izabella, a great app you wrote all ready, I am impressed by your DB handling. Wish you all the best!

File: components/Card/index.js (hunk 2 above)
Comment (izabelladobielewska) [reply components/Card/index.js:98]: awww thanksss!

File: components/Card/index.js (hunk 1 above)
Comment (izabelladobielewska) [reply components/Card/index.js:5]: 😎

File: components/Header/index.js (hunk 3 above)
Comment (izabelladobielewska) [reply components/Header/index.js]: the DB costed me 7 days and many tantrums 😅

**Pull request merged**